app/controllers/students/tests_controller.rb

```diff
@@ -1,8 +1,17 @@
 class Students::TestsController < ApplicationController
 
+  def index
+    @test_setting = TestSetting.find(params[:test_setting_id])
+    @questions = @test_setting.questions
+    tests_belongs_to_test_setting = Test.where(test_setting_id: @test_setting.id)
+    @attempts_number = tests_belongs_to_test_setting.where(user_id: current_user.id)
+    @test = current_user.tests.last
```

Comment:
I redirect to index page bacause I need test_id in order to insert it into url in coffee file. I don't know  where to get test_id in coffee file https://github.com/begin29/schoolTest/pull/21#discussion-diff-53752368R4